lib/rules/no-useless-catch.js

```diff
@@ -43,6 +46,12 @@ module.exports = {
 						context.report({
 							node,
 							messageId: "unnecessaryCatchClause",
+							suggest: [
```

Comment:
The accepted request is to add autofix, not suggestions?

Reply:
Oh! my bad. will update.